core/Piranha.Manager/Services/ContentUtils.cs

```diff
@@ -61,6 +62,7 @@ public static IList<FieldModel> GetBlockFields(Block block)
                         field.Meta.Name = !string.IsNullOrWhiteSpace(attr.Title) ? attr.Title : field.Meta.Name;
                         field.Meta.Placeholder = attr.Placeholder;
                         field.Meta.IsHalfWidth = attr.Options.HasFlag(FieldOption.HalfWidth);
+                        //field.SortOrder = attr.SortOrder;
```

Comment:
If this is not used it should be removed

Reply:
Fixed